dist/morphdom.js

```diff
@@ -9,6 +9,9 @@ function morphAttrs(fromNode, toNode) {
     var fromValue;
 
     // update attributes on original DOM element
+    if (!toNodeAttrs) {
```

Comment:
```suggestion
    if (!toNodeAttrs || !fromNodeAttrs) {
```

Reply:
This is better place for bail out - because later in loops we both check to and from node attributes, and any of them can be missing.